src/bin/cql-stress-cassandra-stress/settings/command/read_write.rs

```diff
@@ -0,0 +1,437 @@
+use crate::settings::{
+    param::{ParamsParser, SimpleParamHandle},
+    ParsePayload,
+};
+use anyhow::Result;
+
+use super::{Command, CommandParams};
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct Uncertainty {
+    pub target_uncertainty: f32,
+    pub min_uncertainty_measurements: u64,
+    pub max_uncertainty_measurements: u64,
+}
+
+impl Uncertainty {
+    pub fn new(
+        target_uncertainty: f32,
+        min_uncertainty_measurements: u64,
+        max_uncertainty_measurements: u64,
+    ) -> Self {
+        Self {
+            target_uncertainty,
+            min_uncertainty_measurements,
+            max_uncertainty_measurements,
+        }
+    }
+
+    pub fn print_settings(&self) {
+        println!("  Target Uncertainty: {}", self.target_uncertainty);
+        println!(
+            "  Minimum Uncertainty Measurements: {}",
+            self.min_uncertainty_measurements
+        );
+        println!(
+            "  Maximum Uncertainty Measurements: {}",
+            self.max_uncertainty_measurements
+        );
+    }
+}
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum Truncate {
+    Never,
+    Once,
+    Always,
+}
+
+impl Truncate {
+    fn parse(truncate: &str) -> Self {
+        match truncate {
+            "never" => Self::Never,
+            "once" => Self::Once,
+            "always" => Self::Always,
+            _ => panic!("Invalid truncate type: {truncate}"),
+        }
+    }
+
+    fn show(&self) -> &'static str {
+        match self {
+            Self::Never => "NEVER",
+            Self::Once => "ONCE",
+            Self::Always => "ALWAYS",
+        }
+    }
```

Comment:
> Another question: why `parse` matches on lowercase, but `show` generates an uppercase string?

Since c-s CLI is case insensivite, I convert everything to lowercase before parsing (and so in the parsing method I assume that everything is in lowercase). But from what I've seen, the enum types such as `Truncate` or `ConsistencyLevel` are usually passed in upper case, that's why i print them like this.